### PR TITLE
uploading /MI-10-MEAN/Pratiksha/Day2.html

### PR DESCRIPTION
**Pull request by pratikshaghule**

### CODE REVIEW

File: Pratiksha/html/Day2.html [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+        <h1>Initial public offering</h1>
+            Google's initial public offering (IPO) took place five years later, on 19th August 2004.
+            At that time Larry Page, Sergey Brin, and Eric Schmidt agreed to work together at Google for 
+            <strike>30</strike> <u>20</u>
+            years, until the year 2024.<br>
+            Thank you!!
+    </p>
+    <div background color:"pink">
+       <h1 align="center"> GOOGLE</h1>
+          <p >This article is about the company.</p>
```
Comment (VIP5614) [inline Pratiksha/html/Day2.html:43]: italic hoga

File: Pratiksha/html/Day2.html [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+          <h1>From the garage to the Googleplex</h1>
+        <p> 
+            <font size="5">Google story begins in 1995 at <b>Stanford University.</b><br>
```
Comment (VIP5614) [inline Pratiksha/html/Day2.html:46]: font normal hoga

File: Pratiksha/html/Day2.html [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+             year they struck a <mark>partnership</mark><br>
+             Working from their dorm rooms, they built a search engine that used links to determine the importance of
+             individual pages on the <span> World Wide Web </span></br>
+             They called this search engine <em style="background-color: pink;">Backrub</em>.</p>
+              <hr>
+
+
+
+              <h1>Initial public offering</h1>
+             Google's initial public offering (IPO) took place five years later, on 19th August 2004.
```
Comment (VIP5614) [inline Pratiksha/html/Day2.html:59]: 19<sup>th</sup>

File: Pratiksha/html/Day2.html [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+             At that time Larry Page, Sergey Brin, and Eric Schmidt agreed to work together at Google for 
+             <strike>30</strike> <u>20</u>
+             years, until the year 2024.<br>
+             <font style="color:green;"><center>Thank you!!</center></font>
```
Comment (VIP5614) [inline Pratiksha/html/Day2.html:63]: font ko bold kar ke font size inures kar dijiye ya h use kariya

**Review by komal453 — APPROVED** [review]

**Review by Snehapadvi96 — APPROVED** [review]

**Pull request merged**